src/h5web/dimension-mapper/SlicingSlider.tsx

```diff
@@ -17,19 +17,15 @@ interface Props {
 function SlicingSlider(props: Props) {
   const { dimension, slicingIndex, rawDims, mapperState, onChange } = props;
 
-  const [containerRef, { height }] = useMeasure();
+  const [containerSize, containerRef] = useMeasure<HTMLDivElement>();
```

Comment:
`@react-hookz/web`'s `useMeasure` puts the ref second and returns an `undefined` rect (whereas `react-use`'s `useMeasure` used to return `{ width: 0, height: 0, ... }`

Reply:
Good, I prefer the `undefined`. It feels clearer me.